changelog.md

```diff
@@ -14,12 +14,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - [[#3233] Add Service Level Interceptor Support for GraphQL Subscriptions](https://github.com/ballerina-platform/ballerina-standard-library/issues/3233)
 
 ### Fixed
-- [[#3294] Fix GraphQL dynamic listener is not working for module level service declaration](https://github.com/ballerina-platform/ballerina-standard-library/issues/3294)
-- [[#3375] Fix Multiple Subscription Endpoints with the Same GraphQL Listener is not Working](https://github.com/ballerina-platform/ballerina-standard-library/issues/3375)
+- [[#3294] Fix GraphQL Dynamic Listener Is Not Working for Module Level Service Declaration](https://github.com/ballerina-platform/ballerina-standard-library/issues/3294)
+- [[#3375] Fix Multiple Subscription Endpoints With the Same GraphQL Listener Is Not Working](https://github.com/ballerina-platform/ballerina-standard-library/issues/3375)
```

Comment:
Prepositions should not be capitalized
```suggestion
- [[#3375] Fix Multiple Subscription Endpoints with the Same GraphQL Listener is Not Working](https://github.com/ballerina-platform/ballerina-standard-library/issues/3375)
```

Reply:
For AP/APA style title case  `With` and `Is` are capitalized.
Are we using a specific format here https://titlecaseconverter.com/ ?

Reply:
For the title case, the Ballerina team practice is not to capitalize the prepositions AFAIK.